src/presentation/routers/article-router.ts

```diff
@@ -0,0 +1,33 @@
+import express, { Request, Response } from 'express';
+import NoSQLWrapper from '../../data/interfaces/data-sources/no-sql-wrapper';
+
+const ArticleRouter = (db: NoSQLWrapper) => {
+    const router = express.Router();
+    router.get('/articles/recent', async (req: Request, res: Response) => {
+    try {
+    const page = parseInt(req.query.page as string) || 1;
+    const pageSize = parseInt(req.query.pageSize as string) || 10;
+    const recentArticles = await getRecentArticlesMock(page, pageSize);
+    res.json(recentArticles);
+    } catch (error) {
+    console.error(error);
+    res.status(500).json({ error: 'Internal Server Error' });
```

Comment:
Organizing response messages in a separate file is a great way to manage and maintain your code, especially in a TypeScript project. Here's an approach you can use:

1. In a separated file you should create /src/busines/messages.ts 
2. Organize your messages in a way that makes sense for your application. You could categorize them based on features, error types, etc
3. Import the messages into the files where you need them and use them as required.
```
// messages.ts

const errorMessages = {
    internalError: 'Internal Server Error'
    // ... other authentication related messages
};


export { authMessages, userMessages };
```
How you can use.

```
response.status(HttpStateCodes.UNAUTHORIZED).json({errorMessages.internalError});
```